docs/user/localization.rst

```diff
@@ -1,17 +1,25 @@
 Localization of Documentation
 =============================
 
-.. note:: This feature only applies to Sphinx documentation. We are working to bring it to our other documentation backends.
+In this article, we explain high-level approaches to internationalizing and localizing your documentation.
 
-Read the Docs supports hosting your docs in multiple languages.
-There are two different things that we support:
+By default, Read the Docs assumes that your documentation is or *might become* multilingual one day.
+The initial default language is English and
+therefore you often see the initial build of your documentation published at ``/en/latest/``,
+where the ``/en`` denotes that it's in English.
+By having the ``en`` URL component present from the beginning,
+you are ready for the eventuality that you would want a second language.
 
-* A single project written in another language
-* A project with translations into multiple languages
+Read the Docs supports hosting your documentation in multiple languages.
```

Comment:
```suggestion
Read the Docs supports hosting your documentation in multiple languages.
Read below for the various approaches that we support.
```

A bit short and not fully explained?

Reply:
Yes, it's nice with the extra sentence here :100: